pkg/controller/migstorage/migstorage_controller.go

```diff
@@ -107,26 +108,33 @@ func (r *ReconcileMigStorage) Reconcile(request reconcile.Request) (reconcile.Re
 	err = r.Get(context.TODO(), request.NamespacedName, storage)
 	if err != nil {
 		if errors.IsNotFound(err) {
+			log.V(2).Info("unable to get migstorage, not found", "migstorage", request.String(), "error", err.Error())
```

Comment:
The `IsNotFound` naturally occurs when the reconcile event has been queued for a deleted resource.  This is worded like an error.  Does logging this really add value?